run.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# Pass "dev" as an argument to run the dev server
+
+# rebuild container
+podman image build -t play-reading . ;
+
+# run the container
+podman run -v play_volume:/var/lib/postgresql/data \
+--name postgres-db \
+-e POSTGRES_PASSWORD=pass \
+-e POSTGRES_USER=playreading \
+-e POSTGRES_DB=playreadingparty \
+-p 5432:5432 \
+-d play-reading
+
+function cleanup {
+  echo "Removing postgres-db container"
+  podman stop postgres-db
+  podman rm postgres-db
+}
+trap cleanup EXIT
```

Comment:
bash `trap`s run a function when the script exits, such as when you press `ctrl + c`